week2/week_2_step_0/.metadata

```diff
@@ -4,7 +4,5 @@
 # This file should be version controlled and should not be manually edited.
 
 version:
-  revision: d954ae6850a06ea461d5595cef424e278ed9f17f
-  channel: dev
-
-project_type: application
+  revision: 5ab9e70727d858def3a586db7fb98ee580352957
+  channel: beta
```

Comment:
You said up in the README.md diff that you were tracking dev in this sample set, yet this is beta. Want to switch to dev to be consistent with the week 1 and week 3 samples?